webpack/assets/javascripts/react_app/components/notifications/NotificationDrawerToggle.js

```diff
@@ -23,10 +24,18 @@ class NotificationDrawerToggle extends Component {
   onChange(actionType) {
     switch (actionType) {
       case ACTIONS.RECEIVED_NOTIFICATIONS: {
-        this.setState({
-          count: NotificationsStore.getNotifications().length,
-          isLoaded: true
-        });
+        const notifications = NotificationsStore.getNotifications();
+
+        const newState = _.reduce(notifications, (curr, next) => {
```

Comment:
@gailsteiger i agree, we're a bit lost on testing UI components with flux, do you have any idea how to do this?
in redux it would much simpler as the store is injected into the component, so different data can be mocked in the store.
in flux currently nothing is injected so there is much less control.